packages/local_auth/local_auth_platform_interface/lib/default_method_channel_platform.dart

```diff
@@ -0,0 +1,65 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/services.dart';
+import 'package:local_auth_platform_interface/local_auth_platform_interface.dart';
+import 'package:local_auth_platform_interface/types/auth_options.dart';
+import 'package:local_auth_platform_interface/types/biometric_type.dart';
+
+const MethodChannel _channel = MethodChannel('plugins.flutter.io/local_auth');
+
+/// The default interface implementation acting as a placeholder for
+/// the native implementation to be set.
```

Comment:
Let's add:

```
/// This implementation is not used by any of the implementations in this
/// repository, and exists only for backward compatibility with any
/// clients that were relying on internal details of the method channel
/// in the pre-federated plugin.
```

That way it's very clear why it's here even though we don't use it.